index.html

```diff
@@ -0,0 +1,115 @@
+      <ul class="mobile-nav-box">
+        <li><a href="about.html">About</a></li>
+        <li><a href="#">Program</a></li>
+        <li><a href="#">Join</a></li>
+        <li><a href="#">Sponser</a></li>
+        <li><a href="#">News</a></li>
+        <li><a href="#">CTS Comapaign</a></li>
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <section class="section-hero gen-padding" id="section-hero">
+      <div class="hero-container">  
+        <div class="intro">
+          <h2 class="secondary">"Hello! Green world"</h2>
+          <h1 class="primary">LONDON CLIMATE TECHNOLOGY SHOW 2023</h1>
+        </div>
+        <p class="intro-description normal">An unparalleled platform for climate technology innovators, policymakers and businesse.</p>
+        <p class="date">26 - 27 Sep, 2023</p>
+        <p class="place">@ ExCeL London, UK</p>
+      </div>
+    </section>
+    <section class="section-main-program gen-padding">
+      <h3 class="title">Main Program</h3>
+      <div class="content-container">
+        <div class="program-item">
+          <div class="icon-box"><img src="assets/Lecture.svg" alt="carbon"></div>
+          <h3 class="program-title">Speach</h3>
+          <p class="program-description normal">From CCUS to carbon markets and other carbon tech, The London Climate Technology .</p>
+        </div>
+        <div class="program-item">
+          <div class="icon-box"><img src="assets/CCexibition.svg" alt="carbon"></div>
+          <h3 class="program-title">Exhibation</h3>
+          <p class="program-description normal">Exhibition at the London Climate Technology Show offers the most established and.</p>
+        </div>
+        <div class="program-item">
+          <div class="icon-box"><img src="assets/Forum.svg" alt="carbon"></div>
+          <h3 class="program-title">Conference</h3>
+          <p class="program-description normal">The London Climate Technology Show will host a stellar lineup of speakers including.</p>
+        </div>
+        <div class="program-item">
+          <div class="icon-box"><img src="assets/Workshop.svg" alt="carbon"></div>
+          <h3 class="program-title">Workshop</h3>
+          <p class="program-description normal">Try creating your own work using open source license rather than just watching at it.</p>
+        </div>
+        <div class="program-item">
+          <div class="icon-box"><img src="assets/CCignite.svg" alt="carbon"></div>
+          <h3 class="program-title">CTS Ignite</h3>
+          <p class="program-description normal">Get opportunities to network with affiliates around the world, also after the summit.</p>
+        </div>
+      </div>
+      <button href="#" class="btn-join">Join CTS 2034</button>
+      <a class="btn-see-program display-none" href="#">SEE THE WHOLE PROGRAM</a>
+    </section>
+    <section class="section-speaker gen-padding">
+      <h3 class="title">Featured Speaker</h3>
+      <div class="speaker-container content-container">
```

Comment:
- [ ] I kindly suggest that you add the more button that is displayed underneath the featured speaker section for mobile .

<img width="383" alt="image" src="https://github.com/Gh-Reza/CTS2023/assets/52934128/61628dac-74c5-45ad-a622-56a6da2ad971">